### ISSUE TITLE
A Node.js build show

### ISSUE THREAD

**Issue by nizaroni** [nodeup/contribute#54]
I think it would be really cool to do a show on the work of the [Node.js build](https://github.com/nodejs/build) working group. What does it do? What are the components? What problems has the working group solved and what challenges lie ahead?

@rvagg has hinted multiple times on NodeUp at the the Node.js build and test infrastructure and it's always piqued my interest, being new to CI myself. Specifically, I'd like to know more about their frustration with the CI offerings out there and what the plans are for the working group to improve their own tools. From the [CI section of the README](https://github.com/nodejs/build#ci-software):

> Our ambition is to invest in io.js-specific CI infrastructure and tooling while leaning on existing, proven technologies where appropriate. We hope to slowly replace Jenkins as orchestrator and build-slaves throughout our CI ecosystem.

**Comment by rvagg** [comment]
yeah, I could ramble for a long time on this topic and it'd be an easy show to do.

@orangemocha and @jbergstroem and @misterdjules are obvious panelists
